includes/Modules/Idea_Hub.php

```diff
@@ -106,20 +107,23 @@ final class Idea_Hub extends Module
 	/**
 	 * Post_Idea_Name instance.
 	 *
+	 * @since n.e.x.t
 	 * @var Post_Idea_Name
 	 */
 	private $post_name_setting;
 
 	/**
 	 * Post_Idea_Text instance.
 	 *
+	 * @since n.e.x.t
 	 * @var Post_Idea_Text
 	 */
 	private $post_text_setting;
 
 	/**
 	 * Post_Idea_Topics instance.
 	 *
+	 * @since n.e.x.t
```

Comment:
Nice catch! Let's update these to use the correct version though as `n.e.x.t` would not be accurate.